From 10c3fc28c903ed7a844686005d56c0c40c8537a9 Mon Sep 17 00:00:00 2001
From: Daniel Axtens <dja@axtens.net>
Date: Mon, 6 Sep 2021 15:46:15 +1000
Subject: [PATCH 028/261] powerpc: Drop Open Hack'Ware - remove
 GRUB_IEEE1275_FLAG_NO_ANSI

Open Hack'Ware was the only user.

Signed-off-by: Daniel Axtens <dja@axtens.net>
Reviewed-by: Daniel Kiper <daniel.kiper@oracle.com>
---
 grub-core/term/ieee1275/console.c | 4 +---
 include/grub/ieee1275/ieee1275.h  | 3 ---
 2 files changed, 1 insertion(+), 6 deletions(-)

diff --git a/grub-core/term/ieee1275/console.c b/grub-core/term/ieee1275/console.c
index 87cbc7a1e..255f0567b 100644
--- a/grub-core/term/ieee1275/console.c
+++ b/grub-core/term/ieee1275/console.c
@@ -248,9 +248,7 @@ grub_console_init_lately (void)
 {
   const char *type;
 
-  if (grub_ieee1275_test_flag (GRUB_IEEE1275_FLAG_NO_ANSI))
-    type = "dumb";
-  else if (grub_ieee1275_test_flag (GRUB_IEEE1275_FLAG_CURSORONOFF_ANSI_BROKEN))
+  if (grub_ieee1275_test_flag (GRUB_IEEE1275_FLAG_CURSORONOFF_ANSI_BROKEN))
     type = "ieee1275-nocursor";
   else
     type = "ieee1275";
diff --git a/include/grub/ieee1275/ieee1275.h b/include/grub/ieee1275/ieee1275.h
index 8bf7b10a0..6ce8605d6 100644
--- a/include/grub/ieee1275/ieee1275.h
+++ b/include/grub/ieee1275/ieee1275.h
@@ -105,9 +105,6 @@ enum grub_ieee1275_flag
   /* OLPC / XO firmware hangs when accessing USB devices.  */
   GRUB_IEEE1275_FLAG_OFDISK_SDCARD_ONLY,
 
-  /* Open Hack'Ware don't support the ANSI sequence.  */
-  GRUB_IEEE1275_FLAG_NO_ANSI,
-
   /* OpenFirmware hangs on qemu if one requests any memory below 1.5 MiB.  */
   GRUB_IEEE1275_FLAG_NO_PRE1_5M_CLAIM,
 
-- 
2.36.1

